queue_model_enumerables_v1.py

```diff
@@ -26,8 +26,8 @@
 EDNARefImages = namedtuple('EDNARefImages', ['FOUR', 'TWO', 'ONE', 'NONE'])
 EDNA_NUM_REF_IMAGES = EDNARefImages(0, 1, 2, 3)
 
-CentringMethod = namedtuple('CentringMethod', ['MANUAL', 'LOOP', 'FULLY_AUTOMATIC', "NO"])
-CENTRING_METHOD = CentringMethod(0, 1, 2, 3)
+CentringMethod = namedtuple('CentringMethod', ['NO', 'MANUAL', 'LOOP', 'FULLY_AUTOMATIC', 'XRAY'])
+CENTRING_METHOD = CentringMethod(0, 1, 2, 3, 4)
```

Comment:
In fact it does not make sense to add 'No centering' to the centering methods. There should be a centering or a centering point before each collection entry. If user do not want to center the sample then there is always possibility to deselect the queue entry in the tree.
I reverted back to the previous version.